src/android/PrivacyScreenPlugin.java

```diff
@@ -24,8 +24,26 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
     super.initialize(cordova, webView);
 
     if (preferences.getBoolean("NoAppScreenshots", true)) {
-      Activity activity = this.cordova.getActivity();
-      activity.getWindow().addFlags(WindowManager.LayoutParams.FLAG_SECURE);
+      if (android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
```

Comment:
We can probably get rid of the `if (preferences.getBoolean("NoAppScreenshots", true)) {` check completely now, since it will allow screenshots by default.

Backstory: We had this preference to turn off this feature for internal QA builds, so that we could take screenshots for reporting issues.